core/server/src/main/java/alluxio/master/file/FileSystemMasterClientRestServiceHandler.java

```diff
@@ -251,21 +254,32 @@ public Response free(@QueryParam("path") String path,
   }
 
   /**
-   * @summary get the file descriptors for a path
+   * @summary get the file descriptors for a path.
```

Comment:
style: either `Get the file descriptors for a path.` or `get the file descriptors for a path`